docs/docs/quickstart/react.mdx

```diff
@@ -0,0 +1,243 @@
+        os: 'os-browserify/browser',
+        process: 'process/browser',
+        stream: 'stream-browserify',
+        util: 'util'
+      }
+    },
+    experiments: {
+      asyncWebAssembly: true
+    },
+    plugins: [
+      new webpack.ProvidePlugin({
+        process: 'process/browser',
+        Buffer: ['buffer', 'Buffer']
+      })
+    ]
+  }
+}
+```
+
+Create a `craco.config.js` with the above content, and ensure its at the root of your project.
+
+
+Update the existing calls to react-scripts in the scripts section of your `package.json`` file to use the craco CLI:
+
+```json
+"scripts": {
+  "start": "craco start",
+  "build": "craco build",
+  "test": "craco test"
+}
+```
+
+## Setup
+
+Start by wrapping the `OpenFormatProvider` around your app.
+
+The `<OpenFormatProvider>` is how you initialise the SDK. It should typically wrap your entire application so you can access the various hooks it provides.
+
+As the provider takes care of initialising the SDK for you, you don't have to do much else but you need to pass it a `config` object to specify which network you want to interact with.
+
+The `config` object can be passed through as a `config` prop to the provider.
+
+```tsx
+import { OpenFormatProvider } from '@simpleweb/open-format-react';
+
+function App() {
+  return (
+    <>
+      <OpenFormatProvider config={{ network: 'mumbai' }}>
+        {/* the rest of your app... */}
+      </OpenFormatProvider>
+    </>
+  );
+}
+```
+
+## Reading from the subgraph
+
+There are numerous hooks that you can use to pull data from the [subgraph](/glossary#subgraph). These return a `Query` from `react-query`, please refer to [their documentation](https://tanstack.com/query/v4/docs/guides/queries) for further information.
+
+```tsx
+import { useSaleData } from '@simpleweb/open-format-react';
+
+function MyComponent() {
+  const { data } = useSaleData({ tokenId: '0x...' });
+
+  return <>{data && <pre>{JSON.stringify(data, null, 2)}</pre>}</>;
+}
+```
+
+Or you can make raw requests against the [subgraph](/glossary#subgraph).
+
+```tsx
+import { useRawRequest } from '@simpleweb/open-format-react';
+import { gql } from 'graphql-request';
+
+function MyComponent() {
+  const { data } = useRawRequest({
+    query: gql`
+      {
+        tokens {
+          id
+        }
+      }
+    `,
+  });
+
+  return <>{data && <pre>{JSON.stringify(data, null, 2)}</pre>}</>;
+}
+```
+
+## Connecting to a wallet
+
+Before you can deploy or perform any interactions with the contract you'll want to connect a wallet.
+
+You can allow people to connect their wallets using the `<ConnectButton />` component (which uses [Onboard](https://www.blocknative.com/onboard)) and the `useWallet` hook to get the connection state and the wallet itself if required.
+
+```tsx
+import { ConnectButton, useWallet } from '@simpleweb/open-format-react';
+
+function MyComponent() {
+  const { isConnected, wallet } = useWallet();
+
+  return (
+    <>
```

Comment:
This is quite an important piece of the puzzle, when getting started using the provider you must be connected before being able to use any of the functions extracting from the hooks, is this worth highlighting?

Reply:
I don't think this worth highlighting as connecting your wallet to a dApp is common practice.